helm/superset/values.yaml

```diff
@@ -25,6 +25,11 @@ replicaCount: 1
 # Runn containers as root is not recommended in production. Change this to another UID - e.g. 1000 to be more secure
 runAsUser: 0
 
+# Create custom service account for Superset. If create: true and name is not provided, superset.fullname will be used.
+# serviceAccountName: superset
+serviceAccount:
+  create: false
```

Comment:
How about:
1. Remove "serviceAccountName"
2. creating serviceAccount sestion as you did with two properties:
- create as you did
- name

According to: https://helm.sh/docs/chart_best_practices/rbac/#yaml-configuration

Reply:
I've thought about it. That was even my first choice, but then I've noticed that, there was `ServiceAccountName` variable added in the past #15340, so I've implemented new feature as enhancement beside the current solution.
I don't want to destroy any configs that can exist somewhere.
@mvoitko and @craig-rueda can you give some more feedback about this?

Reply:
Yeps you are right, we would break backward compatibility but on the other site, we could maintain both for sometime and prepare a communication about that right now.

Reply:
If someone have blocked version on helm in their definition it's OK, but when someone is using "latest" it will brake down deployment (or dependent components).
Supporting two solutions for a while should be quite easy (I'll just add one more conditions in `_helpers`) but this could cause some caveats such as overwriting values if `serviceAccountName` and `serviceAccount.name` would be defined in the same time. :thinking:

Reply:
I think it LGTM. If people are using "latest" in any env that matters, that's generally a bad practice. Using the default of `create: false` seems reasonable to me.